go/src/koding/klient/machine/mount/sync/sync.go

```diff
@@ -22,6 +22,10 @@ import (
 // IndexFileName is a file name of managed directory index.
 const IndexFileName = "index"
 
+// DynamicSSHFunc is an adapter that allows to retrieve information needed to
```

Comment:
What about:

```
DynamicSSHFunc locates the remote host which ssh should connect to.
```

Host and port are not the all information needed to make an SSH connection, so the desc may be misleading.